cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -105,6 +114,16 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                 "(default: the empty root path)"
     ).default("").inputGroup()
 
+    private val licenseClassificationsFile by option(
+        "--license-classifications-file",
+        help = "A file containing the license classifications that are used to limit downloads if the included " +
+                "categories are specified in the 'ort.conf' file. If not specified, all data entities are downloaded."
```

Comment:
s/data entities/packages/

"Data entity" is not a wording that we commonly use in ORT (it's unique to the downloader CLI), and we should try to avoid it in user-facing text. If you wanted to emphasize that also project source code could be downloaded, I believe "packages" is still fine. as in ORT speak also projects are (a special kind of) packages.